README.md

```diff
@@ -0,0 +1,146 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building Capsule applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
```

Comment:
we should be linking to the hub.docker in places, aka https://hub.docker.com/orgs/getcapsule, and those images probably.